.travis.yml

```diff
@@ -0,0 +1,31 @@
+env:
+  global:
+    - RUST_BACKTRACE=1
+    - PATH=$PATH:$HOME/.cargo/bin
+os:
+  - linux
+  - osx
+  - windows
+language: rust
+rust:
+  - 1.29.2
+sudo: false
+cache:
+  cargo: true
+before_script:
+  - if ! cargo prune --version | grep -q "0.1.8"; then
+      cargo install cargo-prune --vers="0.1.8" --force;
+    fi
+  - if ! rustfmt --version | grep -q "0.99.1-stable"; then
+      rustup component add rustfmt-preview;
+    fi
+  - if ! cargo clippy --version | grep -q "0.0.212"; then
+      rustup component add clippy-preview;
+    fi
+script:
+  - cargo fmt -- --check &&
+    cargo test --verbose --release &&
+    cargo clippy --verbose --release &&
+    cargo clippy --verbose --release --profile=test
```

Comment:
Again, doing it for all platforms is not going to buy much but rather slow down the build. Can we use the other link i gave you in previous review to do it only for one platform ? Also placing it above the actual binary formation (cargo test) might make more sense as we want to catch the early in the CI and bail out if possible.

Reply:
Not necessarily, windows/mac os/linux code might differ and all variations might have code style issues reported by clippy.

Reply:
We don't have much platform dependent code and it doesn't buy us much to delay the CI. Besides we don't do it in any of our crates (no clippy on anything but Linux). If for the same code clippy produces diff results on different OSes then it's their problem to fix. So far i don't think we have had issues to make us implement clippy checks on all platforms.